src/app-account.js

```diff
@@ -45,7 +45,19 @@ app.post('/login', (req, res) => {
   switch (loginMethod) {
     case 'header': {
       let headerVal = req.get('x-actual-password') || '';
-      console.debug('HEADER VALUE: ' + headerVal);
+      let obfuscated = '*';
+      if (headerVal.length > 4) {
+        let regex = new RegExp('^(.{2}).*(.{2})$', 'g');
+        obfuscated = headerVal.replace(
+          regex,
+          '$1' + '*'.repeat(headerVal.length - 1) + '$2',
+        );
+      } else if (headerVal.length > 0) {
+        obfuscated = headerVal[0] + '*'.repeat(headerVal.length - 1);
+      } else {
+        obfuscated = 'No password provided.';
+      }
+      console.debug('HEADER VALUE: ' + obfuscated);
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏quite a lot of code to obfuscate this thing. Could we simplify it to something much more _stupid_? That would reduce the chances of this code having bugs that we hadn't considered.

```suggestion
      const obfuscated = '*'.repeat(headerVal.length) || 'No password provided.';
      console.debug('HEADER VALUE: ' + obfuscated);
```

Reply:
Just wanted to give some characters to help validate if the received password was valid or not.

But I’m ok with it